### PR TITLE
i created my own branch

### PR DESCRIPTION
**Pull request by Muqoddim-arotayo**

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>This is eniola's branch</h1>
```
Comment (JustFelix-Dev) [inline signup.html]: Welldone,Oya get to the real work

Comment (Muqoddim-arotayo) [reply signup.html]: oppooooor 😁😁😁

**Comment by JustFelix-Dev** [comment]
I have decided to open your pull request again.😊

File: signup.html (hunk 1 above)
Comment (Muqoddim-arotayo) [reply signup.html]: i've successfully created my signup page

Comment (JustFelix-Dev) [reply signup.html]: Beautiful

**Review by JustFelix-Dev — COMMENTED** [review]
Beautiful. I would go through it soon

**Comment by JustFelix-Dev** [comment]
Nice Sign-Up Page, Eniola😉.
Merged your work.

**Comment by Muqoddim-arotayo** [comment]
> Nice Sign-Up Page, Eniola😉. Merged your work.

merged 👽

**Pull request merged**